src/pages-and-resources/discussions/app-config-form/apps/legacy/LegacyConfigForm.jsx

```diff
@@ -34,14 +35,41 @@ Yup.addMethod(Yup.object, 'uniqueProperty', function (propertyName, message) {
   });
 });
 
+// eslint-disable-next-line func-names
+Yup.addMethod(Yup.string, 'compare', function (message) {
+  // eslint-disable-next-line func-names
+  return this.test('isGreater', message, function () {
+    if (!this.parent || !this.parent.startTime || !this.parent.endTime) {
+      return true;
+    }
+    const isStartTimeGreater = moment(`${moment(this.parent.startDate).format('YYYY-MM-DD')}T${this.parent.startTime}`)
+      .isSameOrAfter(moment(`${moment(this.parent.endDate).format('YYYY-MM-DD')}T${this.parent.endTime}`));
+
+    if (isStartTimeGreater) {
+      throw this.createError({
+        path: `${this.path}`,
+        error: message,
+      });
+    }
+    return true;
+  });
+});
+
```

Comment:
move this utility function along with uniqueProperty function to utils file there we have another util function with same name we have to rename this uniqueProperty to something else e.g `uniqueObjectPropert`.  and do something like this https://github.com/edx/frontend-app-course-authoring/blob/0060f157526886dce648a0c3fce02e977aabe97e/src/utils.js#L46